UPGRADE.rst

```diff
@@ -75,6 +75,27 @@ for example:
      wget https://packages.matrix.org/debian/pool/main/m/matrix-synapse-py3/matrix-synapse-py3_1.3.0+stretch1_amd64.deb
      dpkg -i matrix-synapse-py3_1.3.0+stretch1_amd64.deb
 
+Upgrading to v1.19.0
+====================
+
+New HTML templates
+------------------
+
+A new HTML template,
+`password_reset_confirmation.html <https://github.com/matrix-org/synapse/blob/develop/synapse/res/templates/password_reset_confirmation.html>`_,
+has been added to the ``synapse/res/templates`` directory. If you are using a
+custom template directory, you will need to copy the template over and modify
+it if necessary.
```

Comment:
well done for remembering this, but I'm drawing a line. Let's not add any more templates before we fix https://github.com/matrix-org/synapse/issues/6583.

Reply:
in case it wasn't clear: please fix #6583 as a precursor to landing this PR.

Reply:
Good point. I do want this template to be customisable though. I'll look into the work required to fix that.

Reply:
https://github.com/matrix-org/synapse/pull/8037 is a PR to use default templates.

Reply:
https://github.com/matrix-org/synapse/pull/8037 is now merged :tada: